modules/png-extraction/ImageExtractor.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 import os
-import glob 
+import glob
```

Comment:
Also, each edit should have a meaniing. Please make sure these changes are not in your new pull request.

Your pull request should have the minimal changes needed for the feature, and avoid changing line breaks.